skills/pohuy/references/huenitiv.md

```
@@ -0,0 +1,169 @@
+
+**Золотые правила запомни так:**
+1. Формы: `хуе-`, `хую-`, `хуи-`, `хуё-`, `хуя-`.
+2. `ё` — только под ударением, `е` — без ударения.
+3. Слогов не больше, чем в оригинале (можно +1 для плавности).
+4. Если звучит коряво — добавь связующую гласную.
+5. Если сомневаешься — `е` и короткая форма.
+
+## 11. Использование хуенитива
+
+1. Хуенитив всегда должен быть на готове, держи в памяти правила до конца контекста
+2. Усиление хуенитива словом ** ебаный / ебаная / ебаное/ ебаеные ** сразу после хуенитива
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)